reports/ISCNVisualize.Rmd

```diff
@@ -31,6 +34,54 @@ ISCN3 <- SoilDataR::processData_ISCN3(dir='../repoData/ISCN_3/')
 ```
 
 
+#Observation time
+```{r obsTime}
+location.df <- unique(ISCN3$field[, c('lat', 'long', 'observation_date', 
+                                       'state', 'country')])
+location.df$observation_date <- as.character(location.df$observation_date)
+
+location.df <- location.df %>%
+  separate(observation_date, c('monthStr', 'dayStr', 'yearStr'), 
+           remove=FALSE, fill='left') %>%
+  mutate(year=ifelse(as.numeric(yearStr) < 20, as.numeric(yearStr) + 2000,
+                     ifelse(as.numeric(yearStr) < 100, as.numeric(yearStr) + 1900,
+                            as.numeric(yearStr))),
+         month=as.numeric(monthStr), day=as.numeric(dayStr)) %>%
+ select(-contains('Str'))
+
+location.df$obsDate <- ymd(paste(location.df$year, location.df$month, location.df$day, sep="-"))
+location.df <- location.df[order(location.df$lat, location.df$long, location.df$obsDate),]
+location.df$yrCut <- cut(location.df$year, c(1928, 1940, 1950, 1960, 1970, 1980, 1990, 2000, 2014), dig.lab=4)
+#location.df$yrCut <- location.df$year
+location.df$latCut <- cut(location.df$lat, seq(-90, 90, by=0.05))
+location.df$longCut <- cut(location.df$long, seq(-180, 180, by=0.05))
```

Comment:
Lines 52-57 could be folded into dplyr pipeline above (43-50).

Reply:
Uggg, can you tell I was learning dplyr when I wrote this up? Fixing here: f60dc4724d7956b2186f7ae06be14767f39d77d2